src/crypto/index.js

```diff
@@ -809,24 +815,35 @@ Crypto.prototype.decryptEvent = function(event) {
  * Handle the notification from /sync or /keys/changes that device lists have
  * been changed.
  *
- * @param {Object} deviceLists device_lists field from /sync, or response from
+ * @param {Object} syncData Object containing sync tokens associated with this sync
+ * @param {Object} syncDeviceLists device_lists field from /sync, or response from
  * /keys/changes
  */
-Crypto.prototype.handleDeviceListChanges = async function(deviceLists) {
-    if (deviceLists.changed && Array.isArray(deviceLists.changed)) {
-        deviceLists.changed.forEach((u) => {
-            this._deviceList.invalidateUserDeviceList(u);
-        });
-    }
-
-    if (deviceLists.left && Array.isArray(deviceLists.left)) {
-        deviceLists.left.forEach((u) => {
-            this._deviceList.stopTrackingDeviceList(u);
-        });
+Crypto.prototype.handleDeviceListChanges = async function(syncData, syncDeviceLists) {
+    // No point processing device list changes for initial syncs: they'd be meaningless
```

Comment:
initial syncs have empty change lists iirc

Reply:
ah, I suspected they probably did. updated comment to reflect